WordPress/src/main/res/color/reader_follow_button_text.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <selector xmlns:android="http://schemas.android.com/apk/res/android">
     <item android:color="@color/reader_following" android:state_selected="true" />
-    <item android:color="@color/reader_follow" />
+    <item android:color="@color/reader_follow_blue" />
```

Comment:
@theck13  I originally thought that menu icon should stay gray, but I've changed my mind. It's blue now, and I've removed the gray icon resource entirely. So, ready for another round!